components/profile/contactInfoTable/index.jsx

```diff
@@ -313,6 +319,7 @@ export default function ContactTable({ editing, property }) {
 ContactTable.propTypes = {
   editing: T.bool,
   property: T.string.isRequired,
+  typestr: T.string.isRequired,
```

Comment:
Variables should be camelcase (`typeStr`), although this could use improved naming to explain what the variable is used for. `contactInfoType`, maybe?

You can also improve the validation: `contactInfoType: T.oneOf(["email", "phone"]).isRequired`, which will confirm that only a valid value is sent. And while you're at that, those should be constants at the top:

```
export const CONTACT_INFO_TYPE_EMAIL = "email";
export const CONTACT_INFO_TYPE_PHONE = "phone";
```

And then your validation will be: `contactInfoType: T.oneOf([CONTACT_INFO_TYPE_EMAIL, CONTACT_INFO_TYPE_PHONE]).isRequired`, and the code below in `contactInfoTable/index.test.jsx` would import and use those variables.